config/locales/api/en.yml

```diff
@@ -57,6 +57,7 @@ en:
     patients_with_controlled_bp: "Patients visited with BP &lt;140/90"
     patients_with_uncontrolled_bp: "Patients visited with BP &ge;140/90"
     patients_with_no_bp_measure: "Patients with no BP measure"
+    patients_with_missed_visits: "Patients with a missed visit"
```

Comment:
Based on this slack [thread](https://simpledotorg.slack.com/archives/CFHKJ5WJY/p1654844642890819?thread_ts=1652363243.310499&cid=CFHKJ5WJY), we will release this fix without other translations. 

We will merge the other translations once transifex raises a PR for them.